app/controllers/users_controller.rb

```diff
@@ -61,11 +61,15 @@ def admin_redirect
   end
 
   def user_valid_destroy
-    @user.reviews.delete_all
+    @user.reviews.each do |review|
```

Comment:
I think you can leave it as is and silence the Hound. 

ALSO - last nitpick - favor `destroy` over `delete`. `delete` just executes a delete statement in SQL, whereas `destroy` uses the ActiveRecord Lifecycle.